lib/govuk_publishing_components/presenters/step_by_step_nav_helper.rb

```diff
@@ -57,7 +57,7 @@ def list(element)
       def create_list_item_content(link)
         if link[:href]
           @link_index += 1
-          href = link_href(link[:active], link[:href])
+          href = link_href(link[:active], link[:href], "step-by-step-nav=#{@options[:content_id]}")
```

Comment:
probably should change this into a object/array so that you can pass in multiple query parameters